flink-cdc-connect/flink-cdc-pipeline-connectors/flink-cdc-pipeline-connector-mysql/src/test/java/org/apache/flink/cdc/connectors/mysql/source/MySqlDataSourceFactoryTest.java

```diff
@@ -79,6 +80,27 @@ public void testNoMatchedTable() {
                 .hasMessageContaining("Cannot find any table by the option 'tables' = " + tables);
     }
 
+    @Test
```

Comment:
We still need a test case for validating if excluding part of tables (not all) works as expected